### ISSUE TITLE
[WASM] Unable to scroll through `AutoSuggestBox` items

### ISSUE THREAD

**Issue by morning4coffe-dev** [unoplatform/uno#11648]
### Current behavior

When trying to scroll through items with arrow keys in the AutoSuggestBox on wasm the suggestion popup automatically closes. This is not the case on uwp. 

https://user-images.githubusercontent.com/78549750/224488097-c8b27799-09f2-4312-8cd2-78acf2542a6d.mp4

### Works on UWP/WinUI

Yes

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022
